inc/Optimization/ServiceProvider.php

```diff
@@ -56,12 +56,8 @@ public function register() {
 			->withArgument( $this->getContainer()->get( 'buffer_optimization' ) );
 		$this->getContainer()->add( 'cache_dynamic_resource', 'WP_Rocket\Optimization\Cache_Dynamic_Resource' )
 			->withArgument( $this->getContainer()->get( 'options' ) )
```

Comment:
We can do `$options = $this->getContainer()->get( 'options' )` once, and then use `$options` everywhere for a little perf improvement